### PR TITLE
Fix handling of OP9 and fix token not beeing send on resume

### PR DESCRIPTION
**Pull request by Plerx2493**
# Summary
- Handling of OP9: The docs specify that you should reconnect on op 9 which we didnt do, formerly we just redo the auth on the same connection
- Resume payload: We used `Utilities.GetFormattedToken(DiscordClient)` which takes the token out of the DiscordConfig but our new system doesnt store that in this place anymore

### CODE REVIEW

**Review by VelvetToroyashi — APPROVED** [review]

File: DSharpPlus/Clients/DiscordClient.WebSocket.cs [hunk 1]
```diff
@@ -250,14 +251,14 @@ internal async Task OnInvalidateSessionAsync(bool data)
         if (data)
         {
             this.Logger.LogTrace(LoggerEvents.WebSocketReceive, "Received INVALID_SESSION (OP9, true)");
-            await Task.Delay(6000);
-            await SendResumeAsync();
+            await InternalReconnectAsync(code: 1000, message: "OP9 acknowledged");
```
Comment (akiraveliara) [inline DSharpPlus/Clients/DiscordClient.WebSocket.cs:254]: are we still resuming here

Comment (Plerx2493) [reply DSharpPlus/Clients/DiscordClient.WebSocket.cs:254]: yes, reconnect looks at the sessionID and if its present we resume

Comment (Plerx2493) [reply DSharpPlus/Clients/DiscordClient.WebSocket.cs:254]: we only not resume on reconnect if the sessionid was set to null prio to reconnecting

Comment (Plerx2493) [reply DSharpPlus/Clients/DiscordClient.WebSocket.cs:254]: To be more specific, we look at it when we get `HELLO`

**Pull request merged**